README.md

```diff
@@ -2,7 +2,23 @@
 
 A project based learning activity for people who are getting started with Git and GitHub.
 
-You can play the game [here](https://githubschool.github.io/github-games/)
+
+To play the game:
+1. Go to the **Settings** tab of this repository.
+1. Scroll down to the section titled _GitHub Pages_
+1. Select **Master** from the Source drop-down.
+1. Click **Save**.
+1. Navigate to the URL provided in the same section.
+
+### Instructions for playing the game
+
+1. Press the space bar to begin.
```

Comment:
Left reviews and line comments
    Created a file and made a commit
    Opened a pull request